src/Tickets/Commerce/Order_Modifiers/Modifier_Admin_Handler.php

```diff
@@ -69,8 +73,8 @@ public static function get_page_slug(): string {
 	 * @return void
 	 */
 	public function register(): void {
-		add_action( 'admin_menu', [ $this, 'add_tec_tickets_order_modifiers_page' ], 15 );
-		add_action( 'admin_init', [ $this, 'handle_delete_modifier' ] );
+		add_action( 'admin_menu', fn() => $this->add_tec_tickets_order_modifiers_page(), 15 );
+		add_action( 'admin_init', fn() => $this->handle_delete_modifier() );
```

Comment:
Be aware that in arrow functions you cant return void. You should always return something, even null :D